libs/langchain/langchain/document_loaders/google_speech_to_text.py

```diff
@@ -0,0 +1,142 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, List, Optional
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+from langchain.utilities.vertexai import get_client_info
+
+if TYPE_CHECKING:
+    from google.cloud.speech_v2 import (
+        RecognitionConfig,
+        SpeechClient,
+    )
+    from google.protobuf.field_mask_pb2 import FieldMask
+
+
+class GoogleSpeechToTextLoader(BaseLoader):
+    """
+    Loader for Google Cloud Speech-to-Text audio transcripts.
+
+    It uses the Google Cloud Speech-to-Text API to transcribe audio files
+    and loads the transcribed text into one or more Documents,
+    depending on the specified format.
+
+    To use, you should have the ``google-cloud-speech`` python package installed.
+
+    Audio files can be specified via a Google Cloud Storage uri or a local file path.
+
+    For a detailed explanation of Google Cloud Speech-to-Text, refer to the product
+    documentation.
+    https://cloud.google.com/speech-to-text
+    """
+
+    _client: SpeechClient
+    _recognizer_path: str
```

Comment:
BaseLoader isn't a pydantic object, are these intended to be class attributes?

Reply:
Ah, I didn't realize that difference. I was following the same pattern as the Vertex AI Search retriever.

I'll remove these extra attributes listed here.